src/ZivoeYDL.sol

```diff
@@ -247,14 +247,13 @@ contract ZivoeYDL is Context, ReentrancyGuard {
                 emit YieldDistributedSingle(distributedAsset, _recipient, _protocol[i]);
             }
             else if (_recipient == IZivoeGlobals_YDL(GBL).stZVE()) {
-                uint256 splitBIPS = (
-                    IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply() * BIPS
-                ) / (
-                    IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply() + 
-                    IERC20(IZivoeGlobals_YDL(GBL).vestZVE()).totalSupply()
-                );
-                uint stZVEAllocation = _protocol[i] * splitBIPS / BIPS;
-                uint vestZVEAllocation = _protocol[i] * (BIPS - splitBIPS) / BIPS;
+
+                uint stZVEAllocation = _protocol[i] * (IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply() * BIPS) /
```

Comment:
BIPS on numerator and denominator can be cancelled out

Reply:
Would recommend this which is more readable, and saves some gas due to avoiding repeated totalSupply() calls:

```diff
-                uint stZVEAllocation = _protocol[i] * (IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply()) /
-                    (IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply() + 
-                    IERC20(IZivoeGlobals_YDL(GBL).vestZVE()).totalSupply());
-                uint vestZVEAllocation = _protocol[i] * (IERC20(IZivoeGlobals_YDL(GBL).vestZVE()).totalSupply() * BIPS) /
-                    (BIPS * (IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply() + 
-                    IERC20(IZivoeGlobals_YDL(GBL).vestZVE()).totalSupply()));

+               uint stZVESupply = IERC20(IZivoeGlobals_YDL(GBL).stZVE()).totalSupply();
+               uint vestZVESupply = IERC20(IZivoeGlobals_YDL(GBL).vestZVE()).totalSupply();

+               uint stZVEAllocation = (_protocol[i] * stZVESupply) / (stZVESupply + vestZVESupply);
+               uint vestZVEAllocation = (_protocol[i] * vestZVESupply) / (stZVESupply + vestZVESupply);
```

Reply:
The BIPS in the numerator and denominator would temporarily increase precision, no?